microcosm_pubsub/fields.py

```diff
@@ -1,9 +1,11 @@
 """
 Custom fields.
+
 """
-from marshmallow.fields import Field, ValidationError
 from uuid import UUID
 
+from marshmallow.fields import Field, ValidationError
+
```

Comment:
No, I'm just fixing up a `PEP8` violation; built-ins like `uuid` should come first.